scipy/optimize/_qap.py

```diff
@@ -13,26 +13,24 @@ def quadratic_assignment(
     Solve the quadratic assignment problem.
 
     This function solves the Quadratic Assignment Problem (QAP) and the
-    Graph Matching Problem through an implementation of the Fast
-    Approximate QAP Algorithm (FAQ) (these two problems are the same up
-    to a sign change) [1]_.
+    Graph Matching Problem (GMP) using the Fast Approximate QAP Algorithm
+    (FAQ) [1]_.
```

Comment:
The citation [1]_ belongs right after mentioning the algorithm, not after the bit about the sign change. Let's mention the relationship between the two problems later rather than as a parenthetical thought.